logic.js

```diff
@@ -0,0 +1,28 @@
+function getStartingPositions() {
+    return [
```

Comment:
Add TODO TOMS comment!

Reply:
And please, please(!!) - mark in comment all the types (including Knight => 'H')
👯